### PR TITLE
Avi v3: support minimum 1 cpu

### PR DESCRIPTION
**Pull request by ksivask**
relax minimum CPU constraint to 1 cpu.

### CODE REVIEW

File: repo/packages/A/avi/3/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "packagingVersion": "2.0",
+  "name": "avi",
+  "description": "Avi Vantage ADC",
```
Comment (ryadav88) [inline repo/packages/A/avi/3/package.json]: Add a link to the `examples` doc at the end of `description`

Comment (ryadav88) [inline repo/packages/A/avi/3/package.json]: @ksivask any update here ?

Comment (ksivask) [inline repo/packages/A/avi/3/package.json]: @ryadav88, i was OOO, will work on this.

**Comment by ksivask** [comment]
@ryadav88  please review, added examples and made changes per review. Sent a PR for examples repo too.

File: repo/packages/A/avi/3/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "packagingVersion": "2.0",
+  "name": "avi",
+  "description": "Avi Vantage ADC. An example for this package's usage can be found at https://github.com/dcos/examples/tree/master/avi",
+  "version": "3.0",
+  "framework": false,
+  "scm": "https://github.com/mesosphere/avi-lb",
+  "maintainer": "siva@avinetworks.com",
+  "tags": ["loadbalancer", "service-discovery", "reverse-proxy", "adc", "analytics", "microservices", "gslb"],
+  "preInstallNotes": "We recommend a minimum of 4 CPUs/12 GB of RAM available for the Avi Controller Service; and 1 CPU/2 GB RAM for Avi ServiceEngine.",
```
Comment (ryadav88) [inline repo/packages/A/avi/3/package.json]: @ksivask any update here?

Comment (ksivask) [inline repo/packages/A/avi/3/package.json]: @ryadav88 I have added the preview note. I am able to launch the service fine with defaults.

File: repo/packages/A/avi/3/config.json [hunk 3]
```diff
@@ -0,0 +1,207 @@
+{
```
Comment (ksivask) [inline repo/packages/A/avi/3/config.json]: [dcos.zip](https://github.com/mesosphere/universe/files/1049962/dcos.zip)
attached a zip'ed mp4 video showing our default deployment on v1.9.
Maybe you are hitting the resource constraints? Do you have atleast 1 node in '*' role with sufficient resources - default is 4cpu, 12G ram, 64G disk.

Comment (ryadav88) [inline repo/packages/A/avi/3/config.json]: yes that was it. Thanks for the update and taking the time for the video.

**Pull request merged**